test_supabase_connection.py

```
@@ -0,0 +1,151 @@
+                    # Extract bucket details
+                    bucket_dict = bucket.__dict__ if hasattr(bucket, '__dict__') else bucket
+                    if isinstance(bucket_dict, dict):
+                        bucket_name = bucket_dict.get("name", "unknown")
+                        is_public = bucket_dict.get("public", False)
+                    else:
+                        bucket_name = getattr(bucket, 'name', 'unknown')
+                        is_public = getattr(bucket, 'public', False)
+                    visibility = "public" if is_public else "private"
+                    print(f"       - {bucket_name} ({visibility})")
+                return True
+        except Exception:
```

Pylint (4.0.4)

[warning] Catching too general exception Exception

(W0718)